README.md

```diff
@@ -15,15 +15,41 @@ You need to install Rust and some dependencies.
 You can check os dependent installation [here](https://tauri.studio/v1/guides/getting-started/prerequisites/).
 
 ### Setup the project
-Clone the project, go into it and install the Javascript dependencies.  
-```git clone git@github.com:Brawdunoir/dionysos-client.git```  
-```cd dionysos-client```  
-```pnpm i```
+Clone the project
+```
+git clone git@github.com:Brawdunoir/dionysos-client.git
+```
+CD Into the project directory
+```
+cd dionysos-client
+```
+Install dependencies
+```
+pnpm i
+```
 
 ### Dev
-You can now run the app with hot-reloading.  
-```pnpm tauri dev```
+You can now run the app with hot-reloading
+```
+pnpm tauri dev
+```
 
 ### Build
-```pnpm tauri build```  
-Bundle: ```src-tauri/target/release```
\ No newline at end of file
+```
+pnpm tauri build
+```
+Bundle: ```src-tauri/target/release```
```

Comment:
Same here, for consistencies it needs capitalized letters at the start of "titles" above commands